### ISSUE TITLE
Convenient way to focus on one measure

### ISSUE THREAD

**Issue by l2dy** [allegro/turnilo#286]
Sometimes with multiple measures selected, I'd like to clear them all and select just one measure instead. I would have to remove all unwanted measures one by one in Turnilo.

Proposal: dragging a measure onto the visualization replaces all selected measures.

**Comment by mkuthan** [comment]
It will not be consistent with filters and splits behaviour. But it looks reasonable to add new action to the measures selection list (like for dimension splits). Action "+" for adding measure (existing functionality), new action "[]" for replacing already selected measures with the new one.

**Comment by adrianmroz** [comment]
Hm? But dragging Dimension on visualisation will cause to replace all splits with this dimension. I think analogous behaviour with Measure would be consistent.

**Comment by l2dy** [comment]
> Hm? But dragging Dimension on visualisation will cause to replace all splits with this dimension. I think analogous behaviour with Measure would be consistent.

Yes, that's how I came up with the proposal. Adding a new action for replacing measures in the pop-up would be useful too.

**Comment by l2dy** [comment]
> @l2dy - wow, I was not aware that you can drag the dimension onto the visualisation and replace all splits.
> 
> For me it's weird, especially for line chart visualization when split on time dimension is always required. When you drag non time dimension on line chart you will lost existing chart.

Yes, for line charts this "feature" is annoying. But the "[]" action has the same problem.

> Let's back to d&d for measures. Assume that you have two measures on line chart visualisation and drag another one. All measures should be replaced or the single one where you drag the new one?

If I dragged it onto another selected measure, replace that one; if I dragged it to the blank space after selected measures, add it; if I dragged it onto the visualization, replace all selected measures.

> I would opt for disabling d&d onto the visualization at all :)

I'd opt for keeping it. It's the only way to replace the current selection in one click (actually one d&d, but using actions requires two clicks). I can filter, pin, add split/measure or even replace individual ones (functionality these actions don't cover) by dragging it onto the tiles or the pinboard with one less click than using the action menu.
